### DISCUSSION TITLE
Crashing on ue2.5 game

### DISCUSSION THREAD

**Discussion by nielsbbb** [gildor2/UEViewer#270] [Q&A]
Hi 
Im trying to open .u files from a game to change the gameplay but the program always crashes. 
here is the code `Memory: bad allocation size 1048418116 bytes
appMalloc: size=1048418116 (total=0 Mbytes) <- FArray::Empty: -1344287589 x 12 <- FPackageFileSummary::Serialize2 <- FPackageFileSummary<<: Ver=100/124 <- UnPackage::UnPackage: Echelon.u, ver=100/124, game=ue2 <- UnPackage::LoadPackage(info): Echelon.u <- CUmodelApp::ShowPackageUI <- Main: umodel_build=1587`

**Comment by gildor2** [comment]
You didn't tell explicitly, but it seems you're opening Splinter Cell game (I remember it has "Echelon.u"). Please check the compatibility table, some of these games are supported, some - not. Plus, only single-player data is openable.
